src/Utils/mockBack.ts

```diff
@@ -59,6 +59,20 @@ export const mockUser: UserMoneyData = {
       icon: categories[4].icon,
       category: categories[4].name,
     },
+    {
+      name: "Gasoline",
+      quantity: 230,
+      date: "24-11-2022",
+      icon: "",
+      category: "Transport",
```

Comment:
Pay attention with the categories, if we needed we can do an interface for the categories

Reply:
Yes, it could be a nice idea . Will take it. thanks